auth_ldap_attribute_sync/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'LDAP Attribute Sync',
+    'version': '12.0.1.0.0',
+    'category': 'Tools',
+    'website': 'https://github.com/OCA/server-auth',
+    'author':
+        'Braibean Apps (https://brainbeanapps.com), '
```

Comment:
this needs to be a python list, and I think you should remove the URL from the company's name

Reply:
Done